### PR TITLE
chore(Python): add roman number to int

### PR DESCRIPTION
**Pull request by hharrisd**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

This program converts a string that represents a valid roman number to its decimal equivalent.
If the string is not a valid roman number, raises a ValueError exception.

**Other information:**

### CODE REVIEW

File: algorithms/Python/strings/roman-to-int.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+"""
+
+equivalence = {'I': 1, 'V': 5, 'X': 10, 'L': 50, 'C': 100, 'D': 500, 'M': 1000}
+
+def is_valid_roman_string(roman_number: str) -> bool:
+    """Returns True if all characters of the given string are valid roman numbers"""
+    return set(roman_number).issubset(equivalence.keys())
+
+
+def convertion(roman_number: str) -> int:
```
Comment (Arsenic-ATG) [inline algorithms/Python/strings/roman-to-int.py:15]:
```suggestion
def conversion(roman_number: str) -> int:
```
./algorithms/Python/strings/roman-to-int.py:15: convertion ==> conversion

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
looks good, just fix some typos detected by codespel and the PR should be good to go
 
```
./algorithms/Python/strings/roman-to-int.py:15: convertion ==> conversion
./algorithms/Python/strings/roman-to-int.py:34: convertion ==> conversion
```

File: algorithms/Python/strings/roman-to-int.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+    return result
+
+
+def roman_to_int(roman_number: str) -> int | ValueError:
+    """Main function"""
+    if not is_valid_roman_string(roman_number):
+        raise ValueError(f'The string must contain only valid roman numbers: {equivalence.keys()}')
+
+    return convertion(roman_number)
```
Comment (Arsenic-ATG) [inline algorithms/Python/strings/roman-to-int.py:34]:
```suggestion
    return conversion(roman_number)
```

Comment (hharrisd) [reply algorithms/Python/strings/roman-to-int.py]: Thanks for the corrections, I have applied them in a new commit.

**Pull request merged**